proposals/20220919-airflow_orchestration.md

```diff
@@ -0,0 +1,35 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Varun Murthy
+
+**Your email:** murthyvs@google.com
+
+**Your company/organization:** Google Core ML TFX Team (MTV + Seoul)
+
+**Project name:** Airflow Orchestration
+
+## Project Description
+Moving Airflow to TFX add-ons due to decreasing native support.
```

Comment:
Please describe how the move will be done.  Will Airflow support need to be installed into TFX itself, or can it be installed as a separate package?

Reply:
It can we installed and used as a separate package under tfxa/.